### ISSUE TITLE
Feature request: move icons to preferences

### ISSUE THREAD

**Issue by dangh** [nexional/CompareBuff#1]
Hi,

Thank you for this handy plugin.

I would love to change the window/file icon in the quick panel. My font doesn't has the default characters, so it show up as the question mark.

<img width="79" alt="Screenshot 2022-07-25 at 15 23 44" src="https://user-images.githubusercontent.com/1020347/180732301-4a2244d8-3f2d-4b01-ae39-113a75665579.png">

**Comment by nexional** [comment]
Will check further on that

**Comment by nexional** [comment]
I just released v0.2.9 which gives you more control on the icons through settings. You can toggle them through Command Palette as well. Try it out & let me know.

```
    // Icons, you can disable if you see [?] in selection panel
    "icons":
    {
        "enable": true,
        "icon_ellipsis": "…",
        "icon_package": "🗗",
        "icon_recent_files": "🗍",
        "icon_scratch_file": "🗋",
        "icon_valid_file": "🗎",
        "icon_window": "🗔",
    }
```

**Comment by dangh** [comment]
Thank you. It looks good now

<img width="664" alt="Screenshot 2022-07-28 at 16 32 28" src="https://user-images.githubusercontent.com/1020347/181473164-7055df38-8734-43f5-8186-d31607483531.png">
